src/Tests/FlatSharpCompilerTests/TableMemberTests.cs

```diff
@@ -90,6 +91,9 @@ private void RunCompoundTestWithDefaultValue_Bool(string fbsType)
 
         this.RunSingleTest<T>($"{fbsType} = {randomValue.ToString(format, null).ToLowerInvariant()}", hasDefaultValue: true, expectedDefaultValue: randomValue);
         this.RunCompoundTest<T>(fbsType);
+#if NETSTANDARD2_1 || NET5_0_OR_GREATER
+        this.RunSingleTest<NativeArray<T>?>($"[{fbsType}]  ({MetadataKeys.VectorKind}:\"NativeArray\")");
```

Comment:
So, the only test is just that it builds? If we were to supply some mock implementations could we actually test it?

Reply:
Stubbed and tests added.